packages/dialtone-css/lib/build/less/components/link.less

```diff
@@ -127,4 +127,22 @@
             cursor: not-allowed;
         }
     }
+
+    //  $$  MENTION
+    //  ----------------------------------------------------------------------------
+    // Styling specific to mentions such as @brad.paugh. The underline highlighting
+    // is reversed compared to a regular link, and it has a light background.
+    &--mention {
+        color: var(--dt-color-link-primary);
+        line-height: var(--dt-font-line-height-200);
+        text-decoration: none;
+        background-color: hsl(var(--dt-color-purple-400-hsl) / 0.1);
+        border-radius: var(--dt-size-radius-200);
+
+        &:hover {
+            color: var(--dt-color-link-primary);
+            text-decoration: underline;
```

Comment:
Add
```
text-underline-offset: calc(var(--dt-size-border-300) - var(--dt-size-border-100));
text-decoration-thickness: var(--dt-size-border-50);
```

These properties are used for all `<a>`-presenting items. You can find them elsewhere. Should probably be at least _local_ CSS Variables, or maybe Design Tokens later.

Reply:
Looks like this is already set on d-link actually.

Reply:
However... because it's inverted (mention isn't underlined by default) you have be explicit again.